demisto_sdk/runners/playbook_runner.py

```diff
@@ -0,0 +1,112 @@
+import demisto_client
+from demisto_sdk.common.tools import print_error, print_color, LOG_COLORS
+import time
+from demisto_client.demisto_api.rest import ApiException
+
+
+class PlaybookRunner:
+    """PlaybookRunner is a class that's designed to run a playbook in a given Demisto instance.
+    It will decide whether to wait for the playbook to finish its run or just trigger it, according to the 'wait' flag.
+
+    Attributes:
+        base_link_to_workplan (str): the base link to the workplan of the created incident.
+        demisto_client (demisto_client): object for creating an incident in Demisto.
+    """
+
+    def __init__(self, playbook_id: str, url: str, api: str, wait: str, timeout: int):
+        self.playbook_id = playbook_id
+        self.demisto_url = url
+        self.wait = wait == "True"
+        self.api_key = api
+        self.timeout = timeout
+        self.base_link_to_workplan = f'{self.demisto_url}/#/WorkPlan/'
+        self.demisto_client = demisto_client.configure(
+            base_url=self.demisto_url,
+            api_key=self.api_key,
+            verify_ssl=False)
+
+    def run_playbook(self):
+        # type: () -> int
+        """Run a playbook in Demisto.
+
+        Returns:
+            int. 0 in success, 1 in a failure.
```

Comment:
Why not use exceptions to indicate failures?
That is generally the Pythonic best practice

Reply:
because it's a failure in demisto - not in the code itself